modules/client-configuration/pages/client-automating-preparation.adoc

```diff
@@ -0,0 +1,136 @@
+== Preparation
+
+A configured distribution and an autoinstallation profile is required.
+
+.Procedure: Preparing a Distribution
+
+. Provide the files required to start an installation.
+Unpack an installation medium such as a DVD image on your Server.
+It contains the Linux kernel, an initrd, and other files required to boot the OS in installation mode.
+
+. In the {productname} {webui}, navigate to menu:System[Autoinstallation > Distributions].
+
+. In the [guimenu]``Autoinstallable Distributions`` dialog, click [guimenu]``Create Distribution``:
+* In the [guimenu]``Distribution Label`` field, enter a name to identify your autoinstallable distribution.
+* In the [guimenu]``Tree Path`` field, enter the path to an installation tree located on your {productname} Server.
+* Select the matching [guimenu]``Base Channel`` mirrored on the {productname} Server.
+This base channel must represent the distribution you want to install.
+It can be the [guimenu]``Vendor``, [guimenu]``Custom``, or [guimenu]``Cloned Channels``.
+* The [guimenu]``Installer Generation`` should also match.
+* Optionally, you can specify kernel options which should be added when booting this distribution.
+NOTE: There are multiple places where kernel options can be provided. Only add options here that are generic for the distribution.
+. Click btn:[Create Autoinstallable Distribution].
+
+For more information, see xref:reference:systems/autoinst-distributions.adoc[].
+
+
+.Procedure: Preparing a Profile
+
+. In the {productname} {webui}, navigate to menu:System[Autoinstallation > Profiles].
+
+. In the [guimenu]``Autoinstallation Profiles`` dialog, add the profile for your  autoinstallation.
+It can be an ``AutoYaST`` or ``Kickstart`` profile.
+
+. There are two ways to create profiles:
+* Create a ``Kickstart`` profile using a wizard
+* Upload an externally created profile (``Kickstart`` or ``AutoYaST``)
+
+For more information about the ``Kickstart`` wizard, see xref:reference:systems/autoinst-profiles.adoc[].
+
+
+
+== Create a Profile
```

Comment:
Is this section more for "Upload" a profile?

Reply:
yes, sounds better.  I'll submit the change soon.